routes/teamAPI.js

```diff
@@ -0,0 +1,47 @@
+var teamController = require('../controllers/teamController');
+var authenticator = require('../authenticate/authenticator');
+var router = require('express').Router();
+
+
+//body parameters
+//	required:
+//		name:String
+//	optional:
+//		background:String
+//		coach:String (user)
+//		lead:String(user)
+//		member:[String] (user)
+//		project: [String] (project) 暂时未开放
+router.post('/add',authenticator.authenticate,teamController.create);
+
+//query parameters
+//	required:
+//		id:String (idea)
+router.get('/:id',teamController.getTeamById);
+
+//body parameters
+//	required:
+//	optional:
+//		pageNum:Number (default:1)
```

Comment:
why default value is 1?
Does "1" mean the first page? If yes, please make the first page start from 0, API is for programers, not for end user.